cmd/signimage/signimage.go

```diff
@@ -51,38 +51,88 @@ func signFile(filename string, publickey string, privatekey string) error {
 	return nil
 }
 
-func getAuthFromFile(configfile string, repo string) (authn.Authenticator, error) {
+// allSecretFiles  is a comma seperated list of files and directories to search for creds for repo
```

Comment:
why the allSecretFiles is a list of files? It is the value of pullsecret input parameter, and it gets its value from the ImageRepo of the Module, which is a reference to a sing file

Reply:
Because it allows us to pass in multiple secrets. :-)
The most obvious use of this will be in openshift to pass the secret associated with the builder service account alongside ImageRepoSecret ( e.g. ImageRepoSecret to pull from quay.io, the builder secret to push to the internal registry).  The controller code to pass that in would sit with the openshift kmm repo, not here, but allowing allSecretFiles to be a list here means that the signimage code will be common across both the k8s and ocp, and we'll have a single signing image rather than 2 subtly different ones.

Its also useful functionality to have if you want to use signimage within kubernetes but outside of KMM (e.g. an image build pipeline that just publishes to a registry without wanting to modprobe), so its worth supporting here, even if its not used by Kubernetes KMM.  (I also think we should allow ImageRepoSecret support multiple secrets, but that's an argument for another day)